test/parallel/test-child-process-exec-timeout.js

```diff
@@ -30,6 +30,10 @@ cp.exec(cmd, { timeout: 1 }, common.mustCall((err, stdout, stderr) => {
   // See: https://github.com/libuv/libuv/issues/1226
   if (common.isOSX)
     assert.ok(err.signal === 'SIGTERM' || err.signal === 'SIGKILL');
+  else if (common.isFreeBSD)
+    // Triggers SIGSEGV instead as there is no signal handler
+    // support on v8 for this platform
+    console.log('no signal handler support');
```

Comment:
use `common.skip` or `assert.ok(msg)` would be better?

Reply:
ok common.skip sounds good.

Reply:
Sorry to provide contrary advice but, I'd say:

* Don't use `assert.ok(true, msg)` because...why? The `msg` will never be printed. The assertion will always pass. 

* Don't use `common.skip()` because it will skip all the tests currently running in the file. You only want to skip this test.

I think the way you have it now is fine.

Reply:
(I confirmed that we don't do `assert.ok(true...)` anywhere in the tests with `grep -r 'assert.ok(true' test`.)

Reply:
No worries I ll follow :-) so I just need the "marketing" V8 renaming ?

Reply:
@Trott Sorry, just resolved :)